chapters/functions.tex

```diff
@@ -882,10 +882,8 @@ \subsection{Automatic Vectorization}\label{scalar-functions-applied-to-array-arg
   sum1 := e1 + e2;
 end Add;
 \end{lstlisting}
-\lstinline!Add(1, [1,2,3])! adds one to each of the elements of the second
-argument giving the result \lstinline![2,3,4]!. However, it is illegal to
-write \lstinline!1 + [1,2,3]!, because the rules for the built-in
-operators are more restrictive.
+\lstinline!Add(1, [1,2,3])! adds one to each of the elements of the second argument giving the result \lstinline![2,3,4]!.
```

Comment:
Should we take the opportunity to also change to the common convention of lower camel case for function classes, `Add` → `add`?

Reply:
Yes. Changing.

Reply:
OK. Resolving.